templates/code-page.html

```diff
@@ -156,10 +206,18 @@ <h2 class="mt-4">{{assignment_header}}</h2>
           next_assignment }}</button>
         {% endif %}
         {% if prev_level %}
-          <button class="green-btn" onclick="window.location = '{{ hedy_link(prev_level, 1) }}';">{{regress_button}} {{ prev_level }}</button>
+          {% if not adventure_name %}
```

Comment:
Same here and in the diff immediately below. These look like reversions.

Reply:
Fixed!